client/blocks/comment-detail/index.jsx

```diff
@@ -17,6 +17,7 @@ import CommentDetailPost from './comment-detail-post';
 import CommentDetailReply from './comment-detail-reply';
 import { decodeEntities, stripHTML } from 'lib/formatting';
 import { getPostCommentsTree } from 'state/comments/selectors';
+import getSiteComment from 'state/selectors/get-site-comment';
```

Comment:
I'm not sure about the bundle size benefits.
Just done this because it's the only selector we pull from `state/selectors`.